packages/story-editor/src/components/canvas/navLayer.js

```diff
@@ -27,24 +27,29 @@ import Footer from '../footer';
 import DirectionAware from '../directionAware';
 import PageSideMenu from './pageSideMenu';
 import { FooterArea, HeadArea, Layer, PageMenuArea, Z_INDEX } from './layout';
+import { ChecklistCountProvider } from '../checklist';
 
 function NavLayer({ header, footer }) {
   return (
-    <Layer
-      pointerEvents="none"
-      zIndex={Z_INDEX.NAV}
-      onMouseDown={(evt) => evt.stopPropagation()}
+    <ChecklistCountProvider
+      hasChecklist={Boolean(footer?.secondaryMenu?.checklist)}
```

Comment:
Open to suggestions if ya'll have a better/more straightforward way to communicate if the checklist even exists to the story details modal.

Reply:
Looks like this is how we're checking whether or not to render the other checklist button too, so I think this is what we got to work with 🤷

Reply:
Looks like the count provider can be moved above the checklist provider since it doesn't depend on the checklist provider

💭 Is there a way to derive it from the `value` in the `ChecklistCountProvider`?

```js
// packages/story-editor/src/components/checklist/countContext/checkCountContext.js

function ChecklistCountProvider({ children }) {
  const value = useState(INITIAL_STATE);

  // Grab the issues and see if any of the issue types have any issues registered
  const hasChecklist = Object.values(value).some((issues) => issues.length);

  return (
    <CountContext.Provider value={value}>{children}</CountContext.Provider>
  );
}
ChecklistCountProvider.propTypes = {
  children: PropTypes.node,
};
```

But I faintly remember that some issues don't show until certain checkpoints have been passed 🤔. Maybe that would make the above approach not work?